src/Export_Command.php

```diff
@@ -179,8 +179,8 @@ function( $file_path ) {
 				wp_export(
 					[
 						'filters'     => $this->export_args,
-						'writer'      => 'WP_Export_Stdout_Writer',
-						'writer_args' => null,
+						'writer'      => 'WP_Export_File_Writer',
+						'writer_args' => 'php://output'
```

Comment:
```suggestion
						'writer_args' => 'php://output',
```

Reply:
For the record: I initially preferred `php://output` in the hope that I could: `ob_start()` / `ob_get_clean()` in order to fetch the dump in a variable.
Sadly not, because ob would have to be setup after `fopen` and before `fclose`, so unsuitable for my use-case. I end up copy/pasting so the of the `Base Writer` logic but left `php://output` as an historical attempt.